app/assets/stylesheets/users/user-page.scss

```diff
@@ -51,7 +51,7 @@
 }
 
 .interaction-section {
-  text-align: end;
+  // text-align: end;
   font-size: 0.8rem;
-  margin-top: -10px;
+  // margin-top: -10px;
```

Comment:
- Please, kindly remove these commented pieces of code to make your codebase clean.